l5kit/l5kit/random/random_generator.py

```diff
@@ -58,12 +58,13 @@ class GaussianRandomGenerator(RandomGenerator):
     Note that this gaussian can be multidimensional.
     """
 
-    def __init__(self, mean: np.ndarray, std: np.ndarray):
+    def __init__(self, mean: np.ndarray, std: np.ndarray, random_seed: Optional[int] = 0):
```

Comment:
fine with this, although it should still be using the default numpy seed so it's not clear to me how this was creating any issues (multiproc maybe?)

Reply:
But there is a seeding for the workers in the dataloader too, so I'm not sure why was this causing issues too. cc @qianguih

Reply:
in the current implementation, the seed could be reset by any unexpected `np.random.seed` call. in general, it's safer to maintain it's own rng so that nothing else would affect it. check [this blog post](https://albertcthomas.github.io/good-practices-random-number-generators/) for more details.